README.md

```diff
@@ -187,6 +187,21 @@ The cohort endpoints return JSON in this format:
 ## [Register an Application](#register)
 Census uses [Devise](https://github.com/plataformatec/devise), [Doorkeeper](https://github.com/doorkeeper-gem/doorkeeper) and [cancancan](https://github.com/CanCanCommunity/cancancan) to manage authentication and authorization.
 
+To Get a token, register your app through Census and then make a request:
+
+Source [here.](https://github.com/doorkeeper-gem/doorkeeper/wiki/Client-Credentials-flow)
+
+```javascript
+conn.post do |req|
+  req.url '/oauth/token'
+  req.params['grant_type'] = 'client_credentials'
+  req.params['client_id'] = ENV['CENSUS_CLIENT_ID'] #=> To be supplied by the developer
+  req.params['client_secret'] = ENV['CENSUS_SECRET_ID'] #=> To be supplied by the developer
+end
```

Comment:
I intentionally tried to be vague because with this information, any student could generate a token and access the census database. 

I will make it clearer later tonight 👍 @neight-allen